tools/run_tests/run_interop_tests.py

```diff
@@ -173,6 +174,36 @@ def __str__(self):
         return 'csharpcoreclr'
 
 
+class AspNetCoreLanguage:
+
+    def __init__(self):
+        self.server_cwd = '../grpc-dotnet/testassets/InteropTestsWebsite/bin/Debug/netcoreapp3.0'
+        self.safename = str(self)
+
+    def cloud_to_prod_env(self):
+        return {}
+
+    def client_cmd(self, args):
+        # client not supported
+        return ['dotnet', 'exec', 'InteropClient.cll'] + args
```

Comment:
nit: `return ['echo', 'not supported']` ? `InteropClient.cll` isn't actually a client, right?

Reply:
I will replace with something that actually fails - silently passing unimplemented test cases are bad.